js/src/apbct-public--5--external-forms.js

```diff
@@ -292,9 +292,81 @@ window.onload = function() {
         ctProtectExternal();
         catchDynamicRenderedForm();
         catchNextendSocialLoginForm();
+        ctProtectOutsideIframe();
     }, 2000);
 };
 
+/**
+ * Protect forms placed in iframe with outside src
+ */
+function ctProtectOutsideIframe() {
+    let iframes = document.querySelectorAll('iframe');
+    if (iframes.length > 0) {
+        iframes.forEach(function(iframe) {
+            if (iframe.src.indexOf('form.typeform.com') !== -1 ||
+                iframe.src.indexOf('forms.zohopublic.com') !== -1 ||
+                iframe.src.indexOf('link.surepathconnect.com') !== -1
+            ) {
+                ctProtectOutsideIframeHandler(iframe);
+            }
+        });
+    }
+}
+
+let ctProtectOutsideIframeCheck;
+/**
+ * Protect forms placed in iframe with outside src handler
+ * @param {HTMLElement} iframe
+ */
+function ctProtectOutsideIframeHandler(iframe) {
+    let cover = document.createElement('div');
```

Comment:
div with no id is bad practice, isnt it?

Reply:
I think it is normal.